api/proto/teleport/legacy/types/types.proto

```diff
@@ -5367,7 +5367,7 @@ message PluginSpecV1 {
     // Settings for OpenAI plugin
     PluginOpenAISettings openai = 3;
     // Settings for the Okta plugin
-    PluginOktaSettings okta = 4;
+    PluginOktaSettingsV2 okta = 4;
```

Comment:
This is a breaking change. Can we just extend `PluginOktaSettings` instead of making a new type?

Reply:
Yes, should definitely be possible. I wasn't sure what the criteria for up revving the type itself was - but avoiding doing it if possible seems reasonable.